node/src/components/consensus/era_supervisor.rs

```diff
@@ -117,6 +118,8 @@ pub struct EraSupervisor<I> {
     /// TODO: A temporary field. Shouldn't be needed once the Joiner doesn't have a consensus
     /// component.
     is_initialized: bool,
+    /// TODO: Remove.
```

Comment:
Can you add to it that we should remove it once `EraSupervisor` is removed from the `joiner` reactor?